TWNIA2/1. User Guides/3. 登入與檔案傳輸節點.md

```diff
@@ -1 +1,28 @@
+---
+title: 登入與檔案傳輸節點 | en
+tags: Guide, TWNIA2, EN
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# Login node and file transferring node
+
+| Category | Domain Name | Description | Quantity |
+| -------- | -------- | -------- | -------- |
+| Login node     | ln01.twcc.ai     | The following tasks can be performed:<br>1. Submitting and managing jobs<br>2. Accessing files on HFS<br>3. Compiling HPC related applications<br>4. Running the debugger to improve code    |1   |
+| File transferring node     | xdata1.twcc.ai     | The following tasks can be performed:<br>1. Transferring data to/from HFS<br>2. Using SFTP to transfer data  |1    |
+
+
+- The specifications between login node and file transferring node are similar. The two types of nodes have a compatible environment and both provide developing and testing code services.
+
+:::danger
+<i class="fa fa-exclamation-triangle fa-20" aria-hidden="true"></i> **Important:** Please do not perform computing jobs on login node, or the system will be overloaded and crash, then other users will not be able to log in.
+:::
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+TWCC also provides users with the `rsync` command to transfer files, which provides a good experience when transferring large amounts of files. However, the system does not record the transfer of `rsync`, and this may cause security concerns for connot be traced. If you agree to bear the risk and authorize TWCC to enable `rsync` function, please contact <ins><a href="isupport@twcc.ai"></a>isupport@twcc.ai</ins>.
```

Comment:
TWCC also provides users with the `rsync` command to transfer files, which provides a good experience when transferring large amounts of files. However, the system does not record the transfer of `rsync`, and this may cause security concerns for connot be traced. >


TWCC also provides users transferring data with the `rsync` command, which offers a good experience when transferring large amounts of files. However, TWCC does not support recording the transfer log using `rsync,` so you cannot track problems when you have security concerns.